source/mongodump.txt

```diff
@@ -627,17 +627,27 @@ Options
    Creates a file named :file:`oplog.bson` as part of the
    ``mongodump`` output. The ``oplog.bson`` file, located in
    the top level of the output directory, contains oplog entries that
-   occur during the ``mongodump`` operation. This file provides
-   an effective point-in-time snapshot of the state of a
-   ``mongod`` instance. To restore to a specific point-in-time
-   backup, use the output created with this option in conjunction with
-   :option:`mongorestore --oplogReplay`.
+   occur during the ``mongodump`` operation.
+   
+   The ``oplog.bson`` file is a snapshot of the data at the time the
```

Comment:
I think we should stick with the original phrasing of this, " The ``oplog.bson`` file, located in
   the top level of the output directory, contains oplog entries that
   occur during the ``mongodump`` operation." rather than calling it a snapshot. It only captures changes that occur during the mongodump.